wagtailmenus/models/menuitems.py

```diff
@@ -28,7 +31,7 @@ class AbstractMenuItem(models.Model, MenuItem):
     """A model class that defines a base set of fields and methods for all
     'menu item' models."""
     link_page = models.ForeignKey(
-        'wagtailcore.Page',
```

Comment:
Was this change needed? I believe the appname didn't actually change.

Is there a good reason to change away from lazy loading?

Reply:
Hi @Rotund. The model is imported at the top of the file, so I don't see any benefit from lazy loading in this case. I also think it makes the code slightly clearer (we definitely mean the same 'Page' model that is imported and referenced elsewhere in the file)